app/src/main/java/org/fossasia/openevent/app/data/models/Ticket.java

```diff
@@ -19,7 +20,7 @@
     private long quantity;
     private String type;
 
-    @ForeignKey(stubbedRelationship = true)
+    @ForeignKey(stubbedRelationship = true, onDelete = ForeignKeyAction.CASCADE)
```

Comment:
>This will delete the event if ticket is deleted

Why will it delete event if ticket is deleted. Here ticket is child and event is parent. so the action is applied to the child from parent not the other way. Here am I missing something ?

After setting onDelete action, I was able to use `deleteAll` on Event.class.

I didn't actually get your issue https://github.com/Raizlabs/DBFlow/issues/1332. please explain.